README.md

```diff
@@ -65,31 +76,131 @@ Finally, you can run openhexa with
 ./script/openhexa.sh start
```

Comment:
@toch  is there a way that in which I could re-run this health check to see if the application is working?

```bash
[+] Running 7/8
 ✔ Volume "openhexa_minio_data"              Created                                                                                                                                                         0.0s
 ✔ Container openhexa-app-run-710d7e193814   Removed                                                                                                                                                         0.0s
 ⠧ Container app                             Waiting                                                                                                                                                        66.7s
 ✔ Container openhexa-pipelines_scheduler-1  Healthy                                                                                                                                                        21.7s
 ✔ Container openhexa-pipelines_runner-1     Healthy                                                                                                                                                        21.7s
 ✔ Container jupyterhub                      Healthy                                                                                                                                                        11.2s
 ✔ Container openhexa-minio-1                Healthy                                                                                                                                                        36.2s
 ✔ Container frontend                        Healthy                                                                                                                                                         7.1s
application not healthy after 1m0s
```

Reply:
yes, if you have installed it globally

```bash
/usr/share/openhexa/openhexa.sh -g status
```

it'll check that all services are running and that frontend and backend are HTTP-reachable.

Reply:
we solved it together. The user didn't belong to the docker group and couldn't have access to the docker engine.